test/from-draft7/to-2019-09/dependencies-children-has-type-object.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "title": "`dependencies` has properties whose subschema has a type object",
+        "from": {
+            "dependencies": {
+                "foo": { "type": "object" }
+            }
+        },
+        "to": {
+            "dependencies": {
+                "foo": { "type": "object" }
+            },
+            "dependentSchemas": {}
```

Comment:
Can you get rid of empty `dependentRequired`?

Reply:
Done👍